lib/spack/spack/test/builder.py

```diff
@@ -151,6 +151,9 @@ def test_monkey_patching_test_log_file():
     assert builder.pkg_with_dispatcher.tester.test_log_file == "/some/file"
 
 
+# Windows context manager's __exit__ fails with ValueError ("I/O operation
+# on closed file").
+@pytest.mark.skipif(sys.platform == "win32", reason="Does not run on windows")
```

Comment:
(Question) Do we have any idea why this is happening?

Reply:
No.  I don't have a Windows box to test and wasn't getting more from the PR check than what I added to the comment.